Bottom: 84d06ae4a4863af87357b5141f286c3d10659379
Top:    7cd1e1968b74de4f8e719d9092dc9e1ea4591171
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-21 23:23:34 +0000

Refresh of polygon-c-for-some-reason-the

---

diff --git a/src/polygon.c b/src/polygon.c
index 0ce7b4b..4b2cacc 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -483,7 +483,7 @@ frac_circle (PLINE * c, Coord X, Coord Y, Vector v, int fraction)
   e1 = (v[0] - X) * POLY_CIRC_RADIUS_ADJ;
   e2 = (v[1] - Y) * POLY_CIRC_RADIUS_ADJ;
 
-  /* XXX: */ /* NB: the caller adds the last vertex, hence the -1 */
+  /* XXX */ /* NB: the caller adds the last vertex, hence the -1 */
   range = POLY_CIRC_SEGS / fraction;
   for (i = 0; i < range; i++)
     {
